hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -868,6 +868,16 @@ async def _update_order_status(self):
                     f"Unexpected error during processing order status. The Ascend Ex Response: {resp}", exc_info=True
                 )
 
+    def _update_order_after_failure(self, order_id: str, trading_pair: str):
+        order_update: OrderUpdate = OrderUpdate(
+            client_order_id=order_id,
+            trading_pair=trading_pair,
+            update_timestamp=self.current_timestamp,
+            new_state=OrderState.FAILED,
+        )
+        self._in_flight_order_tracker.process_order_update(order_update)
+        self._in_flight_order_tracker.stop_tracking_order(client_order_id=order_id)
```

Comment:
We should not ask the ClientOrderTracker to stop tracking an order from the connector. That will be done by the ClientOrderTracker after processing the update that changes the order status to a final state.

Reply:
✅